service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -53,6 +56,19 @@ public void start() {
 
         Spark.port(port());
 
+        redirect.any("/", "/health", Redirect.Status.MOVED_PERMANENTLY);
```

Comment:
Well, i just wanted to have something on base url at least for us, like for testing. Maybe not to redirect, but just return the same JSON content as from **/health** url?